### ISSUE TITLE
PCE CD Audio Issues in Ys 3 

### ISSUE THREAD

**Issue by dokusaisha** [TASVideos/BizHawk#776]
The game works perfectly until getting to the Tigray Quarry stage's first screen/area transition. The audio fades to silent, bit kicks back in when the game plays any digital samples (the dialogues w/NPCs in the level). Otherwise it stays silent. Ys 1/2 and Exile have similar problems. It happens will all tested CD images, including the one I personally ripped from an original game disc.

**Comment by zeromus** [comment]
Personally ripped disc images are always assumed to be broken.
I can't believe "all tested CD images" have this problem. If you found one or two... maybe
Try "ripping" your discs from another source.

I just checked ys3, went inside the mine, and still had cd audio

**Comment by dokusaisha** [comment]
I've tried the three different ripped CD images (all different rips, not merely mirrored downloads of the same) that I was able to find online, in addition to the one pulled from my original disc (and just to make sure it wasn't a bad rip, I burned the image back to a CD and put it in my PCE, works perfectly, so the image was good). The audio plays fine when you first go into the mine, but like I said, the first transition within the mine will kill the music. Go all the way to the end of the first corridor, and when the next area loads (the big open area with the scaffolding) the music cuts out. Same thing happens in Ys 1/2 and Exile. The music will play fine initially, but if the music is supposed to continue playing the same track through any of these area transitions, it stops.

I'm only insisting because I know that there's something not right here. I don't blame you for thinking people are either being noobs or they're too lazy to figure out their own problems when they show up out of nowhere like I did, but I only posted this after I was completely sure that I'd done everything I could on my end to fix the problem. I love your work, don't wanna waste your time on stupid stuff.
